Wikipedia/UI-V5/WMFAppViewController.m

```diff
@@ -35,6 +55,8 @@ @interface WMFAppViewController ()<UITabBarControllerDelegate>
 
 @property (nonatomic, strong) SessionSingleton* session;
 
+@property (nonatomic, strong) WMFNavigationTransitionController* navigationTransitionController;
```

Comment:
this is actually something that's a bit hacky.. and dangerously stateful.  let's chat on IRC/hangout